homeassistant/components/philips_js/__init__.py

```diff
@@ -38,15 +38,22 @@
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Philips TV from a config entry."""
 
+    system = entry.data.get(CONF_SYSTEM)
     tvapi = PhilipsTV(
         entry.data[CONF_HOST],
         entry.data[CONF_API_VERSION],
         username=entry.data.get(CONF_USERNAME),
         password=entry.data.get(CONF_PASSWORD),
+        system=system,
     )
     coordinator = PhilipsTVDataUpdateCoordinator(hass, tvapi, entry.options)
 
     await coordinator.async_refresh()
+
+    if tvapi.system and tvapi.system != system:
```

Comment:
Maybe store `tvapi.system` in a local variable with a walrus operator since we use the attribute three times?

Reply:
It was a standard object attribute, so i think it just made it less clear. but it's probably somewhat faster on startup.